rules.ini

```diff
@@ -37,6 +37,7 @@ AdventureGameStudio = (?:^|/)(?:AGSteam\.dll|acsetup\.cfg)$
 AGI = (?:^|/)AGIDATA\.OVL$
 Amazon_Lumberyard = ^bootstrap\.cfg$
 Asura = \.asr$
+BabylonJS = babylon
```

Comment:
is this missing a $ at the end ? (looking at the examples you gave, this might be it)

Reply:
yeye i think i just forgot the $, sorry